e2e_test.ts

```diff
@@ -67,6 +67,71 @@ Deno.test("[e2e] GET /account", async () => {
   assertEquals(resp.status, 303);
 });
 
+Deno.test("[e2e] GET /account/manage", async (test) => {
+  const url = "http://localhost/account/manage";
+
+  const resp1 = await handler(new Request(url));
+  assertFalse(resp1.ok);
+  assertFalse(resp1.body);
+  assertEquals(resp1.headers.get("location"), "/signin");
+  assertEquals(resp1.status, 303);
+
+  await test.step("returns HTTP 404 Not Found response if user does not have a stripeCustomerId", async () => {
+    const user = genNewUser();
+    await createUser({ ...user, stripeCustomerId: undefined });
+    const resp = await handler(
+      new Request(url, {
+        headers: { cookie: "site-session=" + user.sessionId },
+      }),
+    );
+
+    assertFalse(resp.ok);
+    assertEquals(resp.status, Status.NotFound);
+  });
+});
```

Comment:
We could also add a test that tests whether the entire handler executes correctly. This would mean creating a Stripe customer. This test step could be ignored if `isStripeEnabled()` is false. This might be tedious, so I'm happy to tackle this in a proceeding PR.

Reply:
> We should add a test step that tests for when `stripe.billingPortal.sessions.create()` is executed. It'll throw, but testing for that expected behaviour will be good.

Would you be concerned at all with having the tests hit Stripe and failing an authentication check.. just from a resource perspective on Stripes side. I doubt it would rise to the level of rate-limiting or whatever, but it made me wonder.

Reply:
Actually, no longer needed, thanks to the mocking tests you added.